### ISSUE TITLE
SSAO Ultra quality setting throws errors

### ISSUE THREAD

**Issue by honix** [godotengine/godot#70874]
### Godot version

v4.0 beta10

### System information

Windows 10, NVIDIA GeForce RTX 2060, video driver: 31.0.15.2756

### Issue description

SSAO Ultra quality setting throws errors. Image seems incorrect.

SSAO High setting. Image looks correct.
![изображение](https://user-images.githubusercontent.com/11596541/210359036-1fe311eb-6f81-4c37-8aeb-60fadbab5ea0.png)

SSAO Ultra setting. Image looks incorrect.
![изображение](https://user-images.githubusercontent.com/11596541/210359122-423fadfd-c00c-4c9a-a3ed-fd5e8eaf97bf.png)

### Steps to reproduce

- Open reproduction project "SSAO ultra"
- Click Play
- See console for errors

### Minimal reproduction project

[SSAO ultra.zip](https://github.com/godotengine/godot/files/10336633/SSAO.ultra.zip)

**Comment by honix** [comment]
SSIL Ultra seems to throw errors too

**Comment by Calinou** [comment]
I can confirm this on 4.0.beta.custom_build.8a98110e3 (Linux, AMD Radeon RX 6900 XT with Mesa RADV):

```
ERROR: Uniforms supplied for set (0):
Set: 1 Binding: 0 Type: Image Writable: Y Length: 1
Set: 1 Binding: 1 Type: CombinedSampler Writable: N Length: 1
Set: 1 Binding: 2 Type: StorageBuffer Writable: Y Length: 0
are not the same format as required by the pipeline shader. Pipeline shader requires the following bindings:
Set: 0 Binding: 0 Type: CombinedSampler Writable: N Length: 1
Set: 0 Binding: 1 Type: Image Writable: Y Length: 1
Set: 0 Binding: 2 Type: UniformBuffer Writable: N Length: 320
Set: 1 Binding: 0 Type: Image Writable: Y Length: 1
Set: 1 Binding: 1 Type: CombinedSampler Writable: N Length: 1
Set: 1 Binding: 2 Type: StorageBuffer Writable: Y Length: 0
Set: 2 Binding: 0 Type: Image Writable: Y Length: 1
   at: compute_list_dispatch (drivers/vulkan/rendering_device_vulkan.cpp:8009)
```

SSAO Ultra Quality looks the same as having SSAO disabled (meshes don't turn black).

SSAO High Quality | SSAO Ultra Quality
-|-
![2023-01-03_23 21 45](https://user-images.githubusercontent.com/180032/210451015-2ac18e63-d6e7-42f8-8c89-e4ccce7955e6.png) | ![2023-01-03_23 20 57](https://user-images.githubusercontent.com/180032/210451020-f989a64a-d15b-4df8-b27f-a4d94f7ac00e.png)

**Comment by BastiaanOlij** [comment]
Fixed this today, please test in #72838
